src/libvmemcache.h

```diff
@@ -80,7 +80,7 @@ extern "C" {
  */
 typedef struct vmemcache VMEMcache;
 
-enum vmemcache_replacement_policy {
+enum vmemcache_repl_p {
```

Comment:
rationale? you are changing public interface...

Reply:
Heh, I'm rewriting the oonfig API, and also named the field `repl_p`.  Łukasz's change seems natural — Cobol Fingers is a nasty disease, let's protect users against it :)

Reply:
It is shorter and this name ("repl_p") is already used in vmemcache_repl.c.
The library has not been released yet, so we can change the public interface.